Automattic/jetpack
# Boost: Add default URLs for foundation pages

## Proposed changes:
* If the site has woocommerce, add the woocommerce URLs as default foundation pages
* If cornerstone content is set in Yoast, use those in foundation pages

### Other information:


## Jetpack product discussion
None

## Does this pull request change what data or activity we track or use?
None

## Testing instructions:
* Install woocommerce and setup
* Install yoast mark a couple of pages as cornerstone content
* Check the foundation pages settings, they should contain those pages as default
